src/main/java/bio/terra/service/snapshotbuilder/utils/SearchConceptsQueryBuilder.java

```diff
@@ -31,33 +31,43 @@ public static String buildSearchConceptsQuery(
     // domain clause filters for the given domain id based on field domain_id
     var domainClause = createDomainClause(conceptTablePointer, conceptTableVariable, domainId);
 
-    // search concept name clause filters for the search text based on field concept_name
-    var searchNameClause =
-        createSearchConceptClause(
-            conceptTablePointer, conceptTableVariable, searchText, "concept_name");
+    // if the search test is empty do not include the search clauses
+    // return all concepts in the specified domain
+    if (searchText.isEmpty()) {
+      Query query =
+          new Query(List.of(nameField, idField), List.of(conceptTableVariable), domainClause, 100);
+      return query.renderSQL(platform);
+    } else {
 
-    // search concept name clause filters for the search text based on field concept_code
-    var searchCodeClause =
-        createSearchConceptClause(
-            conceptTablePointer, conceptTableVariable, searchText, "concept_code");
+      // search concept name clause filters for the search text based on field concept_name
```

Comment:
what's the difference between searchCodeClause and searchNameClause?

Reply:
searchName clause adds a string of sequel that searches for matches in the concept_name field while searchCode clause does the same for searches against the concept_code field. Together his allows users to search by either concept name or concept code!